<?xml version="1.0" ?>
<launch>

<rosparam file= "$(find robot_control_VS)/param/parameters_ur_PVS.yaml" command= "load"/>

<include file="$(dirname)/camera_ur_calibration.launch" />

<!-- <include file="$(find cartesian_controllers_ur_robots)/launch/robot.launch" /> -->
<include file="$(find ur5_moveit_config)/launch/demo.launch"/>

<node name="publish_L515" pkg="robot_control_VS"  type="publish_L515"  output="screen"/> 

<node name="publish_LUCID" pkg="robot_control_VS"  type="publish_LUCID"  output="screen"/> 

<node name="publish_L515_LUCID" pkg="robot_control_VS"  type="publish_L515_LUCID"  output="screen"/> 

<node name="save_pose_image" pkg="robot_control_VS"  type="ur_save_pose_jointangle_image"  output="screen"/>

</launch>